src/actions/actions.js

```diff
@@ -0,0 +1,160 @@
+import axios from 'axios';
+import * as actions from './actionTypes';
+
+const BASE_URL = '';
+const API_URL = '';
+
+//GET ALL POSTS
+export function loadPostsFromAPI() {
+    return async function(dispatch) {
+        const res = await axios.get(API_URL);
```

Comment:
Would be better if API calls are not included with actions.

Can extract API calls out into a separate folder e.g. `/api`

After which you call the APIs using thunks in the actions